lib/mbta_v3_api/store/predictions.ex

```diff
@@ -5,28 +5,153 @@ defmodule MBTAV3API.Store.Predictions do
 
   Based on https://github.com/mbta/dotcom/blob/main/lib/predictions/store.ex
   """
+  use GenServer
+  require Logger
+  alias MBTAV3API.Prediction
+
   @behaviour MBTAV3API.Store
 
+  @predictions_table_name :predictions_from_streams
+
+  @spec start_link(Keyword.t()) :: GenServer.on_start()
+  def start_link(_) do
+    GenServer.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  @impl true
+  def init(_) do
+    _table = :ets.new(@predictions_table_name, [:named_table])
```

Comment:
Did you try making the table name an argument to `start_link/1`/`init/1` and `fetch/1`/`fetch_multi_filter/1`? That feels like it might be nicer than having to mess with the application config to not start the real predictions store, but the fact that `fetch` runs outside the `GenServer` means that the table can't really be anonymous and stored within the `GenServer`, so if it doesn't actually work all that well then the application config solution is definitely acceptable.

Reply:
I did start down that approach but found it clunkier to have to drill down the table name in more places. Testing the `process_` functions would mean also having to take the GenServer name param - ultimately it seemed to mean more points of failure in trying to write a test that works.